documentation/modules/ref-address-space-example-exports.adoc

```diff
@@ -5,7 +5,14 @@
 [id='ref-address-space-example-exports-{context}']
 = Address space example exports
 
-This example shows the format used by the `ConfigMap` and `Secret` export types:
+You can export your address space information using the following three export types:
+* `ConfigMap`
```

Comment:
This unordered list is not rendering in my Asciidoctor browser plug-in. The AsciiDoctor user manual suggests adding a blank line before the first list item but I'm not 100% sure this is required. See https://asciidoctor.org/docs/user-manual/#unordered-lists